### PR TITLE
added gpu_type and device count

### PR DESCRIPTION
**Pull request by ceizner**
Adding gpu_type to report as a tag on every metric and also reporting device count.

### CODE REVIEW

**Comment by guptaNswati** [comment]
Hi, before I review and merge this, can you please sign and send the CLA and signoff the commits.

Thanks

File: exporters/prometheus-dcgm/dcgm-exporter/dcgm-exporter [hunk 1]
```diff
@@ -57,16 +57,22 @@ function metric(name, type, help, value) {
         if (gpu == 0) {
             printf "# HELP dcgm_%s %s\n", name, help > out".swp"
             printf "# TYPE dcgm_%s %s\n", name, type > out".swp"
+          if ( name == "device_count") {
+                printf "dcgm_%s %s\n", name,value > out".swp"
```
Comment (Jeffwan) [inline exporters/prometheus-dcgm/dcgm-exporter/dcgm-exporter]: Looks like it has some syntax errors. @ceizner Can you help update it?

Comment (Jeffwan) [reply exporters/prometheus-dcgm/dcgm-exporter/dcgm-exporter]: Missing `}`

**Comment by guptaNswati** [comment]
Closing for no response. Please re-open when you can sign-off the commits. Thanks.

**Comment by Jeffwan** [comment]
@guptaNswati I will help file a PR to this issue.

**Pull request closed without merging**